solutions/standard/main.tf

```diff
@@ -120,6 +119,65 @@ module "elasticsearch" {
   enable_elser_model            = var.enable_elser_model
 }
 
+#create a service authorization between Secrets Manager and the target service (Elastic Search)
+resource "ibm_iam_authorization_policy" "policy" {
+  count                       = var.skip_es_kms_auth_policy ? 0 : 1
```

Comment:
Is this test as expected? That looks like a KMS property. Is another SM property required? Is a different property skip_all_auth_policy or skip_auth_policies a better solution?

Reply:
Name of the variable was incorrect. Renamed it to "skip_es_sm_auth_policy".

The "skip_es_sm_auth_policy" is only used once inside the ibm_iam_authorization_policy.secrets_manager_key_manager